tamr_unify_client/project/attribute_mapping_collection.py

```diff
@@ -0,0 +1,30 @@
+import requests
+
+from tamr_unify_client import Client
+from tamr_unify_client.auth import UsernamePasswordAuth
+from tamr_unify_client.project.attribute_mapping import AttributeMapping
+
+
+class AttributeMappingCollection(AttributeMapping):
+    def setUp(self):
+        auth = UsernamePasswordAuth("username", "password")
+        self.unify = Client(auth)
+
+    def stream(self, map_url):
+        map_url = map_url
+        all_maps = requests.get(map_url).json()
+        return all_maps
```

Comment:
Take a look at the `BaseCollection` `stream` function and try to emulate that--right now, this is a list of dicts, not a stream of `AttributeMapping`s